lib/site_prism/dsl/builder.rb

```diff
@@ -15,35 +15,33 @@ module DSL
     #
     module Builder
       # Return a list of all mapped items on a SitePrism class instance (Page or Section)
-      # If legacy is set to false (Default) -> @return [Hash]
-      # If legacy is set to true (Old behaviour) -> @return [Array]
-      def mapped_items(legacy: false)
-        return legacy_mapped_items if legacy
-
+      #
+      # @return [Hash]
+      def mapped_items
         @mapped_items ||= { element: [], elements: [], section: [], sections: [], iframe: [] }
       end
 
       private
 
       def build(type, name, *find_args)
-        raise InvalidDSLNameError if ENV.fetch('SITEPRISM_DSL_VALIDATION_ENABLED', 'true') == 'true' && invalid?(name)
+        invalid_element_name if invalid_element_name?(name)
+        invalid_element(name) if find_args.empty?
```

Comment:
change this to something like undefined element

Reply:
This is just the name, not the action, it will still not be permitted

Reply:
Actually I've handled all of this with the method proper. So marking this as "won't do"